src/NzbDrone.Core/MediaFiles/EpisodeImport/ImportApprovedEpisodes.cs

```diff
@@ -132,7 +141,8 @@ public List<ImportResult> Import(List<ImportDecision> decisions, bool newDownloa
 
                     if (newDownload)
                     {
-                        _extraService.ImportEpisode(localEpisode, episodeFile, copyOnly);
+                        scriptImportDecisionInfo.isEpisodeFile = false;
```

Comment:
Extra service should already know that it's not importing an episode file, this shouldn't be mutated and re-used.

Reply:
I suppose if I am already supposed to break up `ScriptImportDecisionInfo` I can also turn this into a parameter to the `TryImport` method instead.